core/src/main/java/ar/com/wolox/wolmo/core/util/KeyboardUtils.kt

```diff
@@ -0,0 +1,55 @@
+package ar.com.wolox.wolmo.core.util
+
+import android.app.Activity
+import android.content.Context
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+import android.widget.EditText
+import androidx.fragment.app.Fragment
+
+class KeyboardUtils {
```

Comment:
We should inject this one as well. Similar to `ImageProvider`. Feel free to ask questions about how to do it. A heads up, you probably should inject the `applicationContext`.

EDIT: I'm now wondering wether we should inject this one since the `context` that is provided in the different methods is not necessarily the `Application`'s. What do you think?